cterasdk/edge/cache.py

```diff
@@ -23,9 +25,63 @@ def force_eviction(self):
         self._gateway.execute("/config/cloudsync", "forceExecuteEvictor", None)
         logging.getLogger().info("Eviction started.")
 
-    def is_enabled(self):
-        return self._gateway.get('/config/cloudsync/cloudExtender/operationMode') == OperationMode.CachingGateway
-
     def _set_operation_mode(self, mode):
         self._gateway.put('/config/cloudsync/cloudExtender/operationMode', mode)
         logging.getLogger().info('Device opreation mode changed. %s', {'mode': mode})
+
+    def get_pinned_folders(self):
+        """
+        Obtain a file system tree object of pinned folders
```

Comment:
Add doc about return: ":return cterasdk.edge.directorytree.DirectoryTree: The file system tree object"

Reply:
Decided to remove **get_pinned_folders**